ambry-account/src/main/java/com/github/ambry/account/mysql/ContainerDao.java

```diff
@@ -83,6 +84,36 @@ public void addContainer(int accountId, Container container) throws SQLException
     }
   }
 
+  /**
+   * Add a containers to the database.
+   * @param accountId the containers's parent account id.
```

Comment:
nit:
1. `Add containers from given account to database in batches`
2. `containers'`